composables/thread.ts

```diff
@@ -0,0 +1,91 @@
+import type { Draft } from '~/types'
+
+/**
+ * The list of draft keys that are part of the thread.
+ * There is a single global thread for the whole app, to keep complexity low.
```

Comment:
Let's see if that's still the case when a draft can include an entire thread. Might be able to move this to the composables state